cql3/restrictions/multi_column_restriction.hh

```diff
@@ -90,6 +90,8 @@ public:
         auto as_pkr = static_pointer_cast<clustering_key_restrictions>(other);
         do_merge_with(as_pkr);
         update_asc_desc_existence();
+        wip_equivalent = ::make_shared<wip::expression>(
+                wip::conjunction{std::vector{wip_equivalent, other->wip_equivalent}});
```

Comment:
You're converting a list to a tree here. Do you convert it back later?

It's sad because the parser provided us with a list and the existing code destroyed the structure, but we can recover it later.

Reply:
Added make_conjunction(), as suggested below.